tensorflow/python/data/ops/dataset_ops.py

```diff
@@ -611,6 +618,13 @@ def from_tensor_slices(tensors):
            [3, 2]], dtype=int32), array([[b'A'],
            [b'B']], dtype=object))
 
+    >>> # `from_tensor_slices` slices the input tensor,
+    >>> # unlike `from_tensors` which retains the shape of input tensor.
+    >>> example = tf.constant([1,2,3])
+    >>> dataset = tf.data.Dataset.from_tensor_slices(example).repeat(1000)
+    >>> np.array(list(dataset1.as_numpy_iterator())).shape
+    (3000,)
```

Comment:
This example doesn't add new information. But since I am adding example in `from_tensors`, I have also added similar example in `from_tensor_slices` so that one can differentiate both.

Reply:
Changed it as described in below comment.